docs/content/node-operation/machine-existing-operator.mdx

```diff
@@ -0,0 +1,98 @@
+
+If you have an existing `consensus` or `collection` node that you registered with Flow Port before Thursday August 19th, you will need to create this Machine Account manually in order to participate in epochs.
+This guide will walk you through creating a Machine Account and getting it set up.
+
+## Downloading Bootstrap Utility
+
+<Callout type="warning">
+  If you have downloaded the bootstrapping kit previously, ensure that you do
+  this step again to get the latest copy of the bootstrapping kit since there
+  have been significant changes to it.
+</Callout>
+
+```shell
+curl -sL -O storage.googleapis.com/flow-genesis-bootstrap/boot-tools.tar
+tar -xvf boot-tools.tar
+```
+
+```shell:title=CheckSHA256
+sha256sum ./boot-tools/bootstrap
+06c0e8a4350a1fb2cc2ece9b62f6537d23902ddd214ff3fe4db58198be265a0e  ./boot-tools/bootstrap
+
+sha256sum ./boot-tools/transit
+e3599147a358539ff45a96af9a1a89568be9ddc5e210d42edd933871781378c9  ./boot-tools/transit
+```
+
+## Generate Machine Account key
+
+You will need to generate a Machine account private key using the `bootstrap` utility.
+
+```shell
+$ ./boot-tools/bootstrap machine-account-key -o ./bootstrap
+```
+
+```shell:title=Example
+$./boot-tools/bootstrap machine-account-key -o ./bootstrap
+<nil> INF generated machine account private key
+<nil> DBG encoded public machine account key machineAccountPubKey=2743786d1ff1bf7d7026d693a774210eaa54728343859baab62e2df7f71a370651f4c7fd239d07af170e484eedd4f3c2df47103f6c39baf2eb2a50f67bbcba6a
+<nil> INF wrote file bootstrap/private-root-information/private-node-info_6f6e98c983dbd9aa69320452949b81abeab2ac591a247f55f19f4dbf0b477d26/node-machine-account-key.priv.json
+<nil> INF machine account public key: 0x2743786d1ff1bf7d7026d693a774210eaa54728343859baab62e2df7f71a370651f4c7fd239d07af170e484eedd4f3c2df47103f6c39baf2eb2a50f67bbcba6a
+
+$tree ./bootstrap/
+./bootstrap
+├── private-root-information
+│   └── private-node-info_ab6e0b15837de7e5261777cb65665b318cf3f94492dde27c1ea13830e989bbf9
+│       ├── node-info.priv.json
+│       └── node-machine-account-key.priv.json
+└── public-root-information
+    ├── node-id
+    └── node-info.pub.ab6e0b15837de7e5261777cb65665b318cf3f94492dde27c1ea13830e989bbf9.json
+
+3 directories, 4 files
+```
+
+## Create Machine Account
+
+You will now need to copy the Machine account public key displayed in the terminal output and head over to [Flow Port](/flow-port/staking-guide/) to submit a transaction to create a Machine Account. For example, from the example above, we would copy `0x2743786d1ff1bf7d7026d693a774210eaa54728343859baab62e2df7f71a370651f4c7fd239d07af170e484eedd4f3c2df47103f6c39baf2eb2a50f67bbcba6a` from this line:
+/```
+<nil> INF machine account public key: 0x2743786d1ff1bf7d7026d693a774210eaa54728343859baab62e2df7f71a370651f4c7fd239d07af170e484eedd4f3c2df47103f6c39baf2eb2a50f67bbcba6a
+/```
```

Comment:
Sorry, you will need to remove the `/` characters here. I suspect this is what is causing the build to fail as well. It seems I cannot suggest any string containing ` ``` ` without exiting the suggestion block!